torch/csrc/distributed/rpc/init.cpp

```diff
@@ -684,6 +690,12 @@ PyObject* rpc_init(PyObject* _unused, PyObject* noargs) {
           (std::vector<WorkerInfo>(TensorPipeAgent::*)() const) &
               TensorPipeAgent::getWorkerInfos,
           py::call_guard<py::gil_scoped_release>())
+      .def(
+          "_get_device_map",
+          (tensorpipe::DeviceMap(TensorPipeAgent::*)(const WorkerInfo& dest)
```

Comment:
This looks OK to me (as it's a private API). I am still curious to see how this is used in RemoteModule. Since this takes `dst` WorkerInfo as an argument, as it mean `RemoteModule` only cares about the device map for a specific worker (maybe the worker where the module lives in?) and won't need to loop over all workers?

Reply:
Just saw the PR on top. Look so, it just need the device map for one worker.